Build/SaveLoadGame.cc

```diff
@@ -219,7 +219,9 @@ BOOLEAN SaveGame(UINT8 const ubSaveGameID, wchar_t const* GameDesc)
 	InitShutDownMapTempFileTest(TRUE, "SaveMapTempFile", ubSaveGameID);
 
 	// Place a message on the screen telling the user that we are saving the game
-	{ UINT16 actual_w;
+	if (gGameOptions.ubGameSaveMode != DIF_DEAD_IS_DEAD) 
```

Comment:
Due to the frequency of saving, the popup message is getting very annoying very quickly.
It is still displayed in the message log.